x/nft/keeper/collection.go

```diff
@@ -18,14 +18,20 @@ import (
 // SetCollection saves all NFTs and returns an error if there already exists
```

Comment:
not sure if the comment should be updated -- before, it returned error if any of the NFTs existed (i.e. `k.MintNFT` returned an err)... now, also if `nft.GetOwner` returns an error

Reply:
Done